.github/workflows/deploy-cf-it.yml

```diff
@@ -140,6 +140,7 @@ jobs:
         credentials: ${{ secrets.DEPLOY_CF_SA_KEY_JSON }}
         event_trigger_type: providers/cloud.pubsub/eventTypes/topic.publish
         event_trigger_resource: ${{ secrets.DEPLOY_CF_EVENT_PUBSUB_TOPIC }}
+        env_vars_file: './tests/env-var-files/test.good.yaml'
```

Comment:
No Integration tests? It looks like the other jobs run e2e tests

Reply:
These are actually the Integration tests (we build dist and use the built action)  however I need to split the [unit](https://github.com/google-github-actions/deploy-cloud-functions/blob/main/tests/cloudFunction.test.ts) from [e2e](https://github.com/google-github-actions/deploy-cloud-functions/blob/main/tests/cloudFunctionClient.test.ts).